Frontend/ui-driver/src/Helpers/Utils.purs

```diff
@@ -602,36 +603,68 @@ getVehicleVariantImage :: String -> String
 getVehicleVariantImage variant =
   let url = getAssetLink FunctionCall
       commonUrl = getCommonAssetLink FunctionCall
-  in case variant of
-      "SEDAN"     -> "ny_ic_sedan_ac," <> commonUrl <> "ny_ic_sedan_ac.png"
-      "SEDAN_TIER" -> "ny_ic_sedan_ac," <> commonUrl <> "ny_ic_sedan_ac.png"
-      "SUV"       -> "ic_suv_ac," <> commonUrl <> "ic_suv_ac.png"
-      "SUV_TIER"  -> "ic_suv_ac," <> commonUrl <> "ic_suv_ac.png"
-      "HATCHBACK" -> "ic_hatchback_ac," <> commonUrl <> "ic_hatchback_ac.png"
-      "HATCHBACK_TIER" -> "ic_hatchback_ac," <> commonUrl <> "ic_hatchback_ac.png"
-      "RENTALS"   -> "ic_rentals," <> commonUrl <> "ic_rentals.png"
-      "INTERCITY" -> "ic_intercity," <> commonUrl <> "ic_intercity.png"
-      "TAXI"      -> "ic_taxi," <> commonUrl <> "ic_taxi.png"
-      "PREMIUM"   -> "ic_cab_premium" <> commonUrl <> "ic_cab_premium.png"
-      "TAXI_PLUS" -> "ny_ic_sedan_ac," <> commonUrl <> "ny_ic_sedan_ac.png"
-      "ECO"       -> "ic_hatchback_ac," <> commonUrl <> "ic_hatchback_ac.png"
-      "COMFY"     -> "ny_ic_sedan_ac," <> commonUrl <> "ny_ic_sedan_ac.png"
-      "AUTO_RICKSHAW" -> 
-        case (getValueFromCache (show DRIVER_LOCATION) getKeyInSharedPrefKeys) of
-          "Hyderabad" -> fetchImage FF_ASSET "ny_ic_black_yellow_auto1"
-          "Chennai"   -> fetchImage FF_ASSET "ny_ic_black_yellow_auto1"
-          "Kochi"     -> fetchImage FF_ASSET "ny_ic_black_yellow_auto1"
-          _           -> fetchImage FF_ASSET "ic_vehicle_front"
-      "BIKE"      -> "ny_ic_bike_side," <> commonUrl <> "ny_ic_bike_side.png"
-      "AMBULANCE_TAXI" -> "ny_ic_ambulance_side," <> commonUrl <> "ny_ic_ambulance_side.png"
-      "AMBULANCE_TAXI_OXY" -> "ny_ic_ambulance_side," <> commonUrl <> "ny_ic_ambulance_side.png"
-      "AMBULANCE_AC" -> "ny_ic_ambulance_side," <> commonUrl <> "ny_ic_ambulance_side.png"
-      "AMBULANCE_AC_OXY" -> "ny_ic_ambulance_side," <> commonUrl <> "ny_ic_ambulance_side.png"
-      "AMBULANCE_VENTILATOR" -> "ny_ic_ambulance_side," <> commonUrl <> "ny_ic_ambulance_side.png"
-      "BIKE_TIER" -> "ny_ic_bike_side," <> commonUrl <> "ny_ic_bike_side.png"
-      "SUV_PLUS"  -> "ny_ic_suv_plus_side," <> commonUrl <> "ny_ic_suv_plus_side.png"
-      "SUV_PLUS_TIER" -> "ny_ic_suv_plus_side," <> commonUrl <> "ny_ic_suv_plus_side.png"
-      _ -> fetchImage FF_ASSET "ic_vehicle_front"
+  in case getMerchant FunctionCall of
+        YATRISATHI -> case spy "varient is" variant of
```

Comment:
We must not use specific `YATRISATHI` checks in such cases because after the sync with main these have been removed already.